src/main/java/servlets/FetchOptionsServlet.java

```diff
@@ -61,7 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       string in the given array is already capitalized, since this is the case
       for search strings, in which all Strings are fully upper-cased, and
       this servlet is being used to fetch those search strings. */
-  public ArrayList<String> titleCaseItems(ArrayList<String> arr) {
+  public static ArrayList<String> titleCaseItems(ArrayList<String> arr) {
     ArrayList<String> arrTitleCase = new ArrayList<String>();
```

Comment:
FYI, you could do something like:
return arr.stream().map(str -> (your function on str)).collect(Collectors.toList());

Please search for "Java stream collect" for more more details.

Reply:
Fixed! Thank you for the pointer :)